libprecompiled/ChainGovernancePrecompiled.cpp

```diff
@@ -273,6 +284,8 @@ int ChainGovernancePrecompiled::grantCommitteeMember(
     if (entries->size() == 0u)
     {  // grant committee member
         result = grantTablePermission(_context, SYS_ACCESS_TABLE, _member, _origin);
+        result = grantTablePermission(_context, SYS_CONFIG, _member, _origin);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'result' is reassigned a value before the old one has been used.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5420864)